digiroad2-api-oth/src/test/scala/fi/liikennevirasto/digiroad2/dataimport/MassTransitStopCsvImporterSpec.scala

```diff
@@ -333,7 +334,7 @@ class MassTransitStopCsvImporterSpec extends AuthenticatedApiSpec with BeforeAnd
     "vyohyketieto" -> ("InfoZone", "InfoZone ready")
   )
 
-  when(mockGeometryTransform.resolveAddressAndLocation(any[Point], any[Int], any[Double], any[String], any[Int], any[Option[Int]], any[Option[Int]])).thenReturn((RoadAddress(None, 0, 0, Track.Unknown, 0) , RoadSide.Right))
+  when(mockGeometryTransform.resolveAddressAndLocation(any[Point], any[Int], any[Double], any[String], any[Int], any[Option[Int]], any[Option[Int]])).thenReturn((digiroad2.RoadAddress(None, 0, 0, Track.Unknown, 0) , RoadSide.Right))
```

Comment:
import vähän outo

import fi.liikennevirasto.digiroad2.RoadAddress pitäis olla niin ei tarvita digiroad2.RoadAddress muotoa